src/types.ts

```diff
@@ -36,3 +36,10 @@ export type TransformEntry = {
 };
 
 export type BuildEntry = BundleEntry | TransformEntry;
+
+export interface BuildOptions {
+  /**
+   * Path to a tsconfig.json file to use for type checking and compilation.
+   */
+  tsconfigPath?: string;
```

Comment:
We can rename to `tsConfig?: boolean | stirng` (to allow enabling without explicit path)